app/views/incomes/AddIncomeCheckYourAnswersView.scala.html

```diff
@@ -50,12 +61,14 @@
         )
 
         @if(viewModel.postConfirmationText.isDefined) {
-            <p class="subsection">@viewModel.postConfirmationText.get</p>
+            <p class="govuk-body subsection">@viewModel.postConfirmationText.get</p>
```

Comment:
do u need subsection?

Reply:
No, I was meant to remove that as I was trying to figure out why a test was failing, will change that now.